### PR TITLE
Update SIOS dataset

### PR DESCRIPTION
**Pull request by IvanaH8**
Proposes new title and adds summary.

There are some other variables collected by the sensor, such as fDOM and Oxygen concentration and saturation, but I cannot find any appropriate CF standard names for those. They are probably not as important as temperature, etc. so I think we can leave them out, what do you think?

Keywords look good to me as they are.

### CODE REVIEW

**Comment by knl88** [comment]
lgtm, @mar1yS what do you think about the titles?

btw, this is just a metadata draft until we can publish the datasets:)

File: dscreator/datasets/timeseries/sios.py [hunk 1]
```diff
@@ -106,6 +106,16 @@ def map_to_cfarray(self, timeseries: NamedTimeseries) -> xr.DataArray:
                         units="S/m",
                     ),
                 )
+            case "fDOMCalib":
+                array = dataarraybytime(
+                    data=timeseries.values,
+                    name="conductivity",
+                    attrs=CFVariableAttrs(
+                        standard_name="concentration_of_colored_dissolved_organic_matter_in_sea_water_expressed_as_equivalent_mass_fraction_of_quinine_sulfate_dihydrate",
```
Comment (IvanaH8) [inline dscreator/datasets/timeseries/sios.py:114]: This standard name should be double-checked @knl88

Comment (IvanaH8) [reply dscreator/datasets/timeseries/sios.py:114]: Shall we still include the conductivity in the dataset? Sabine did not mention conductivity. She did say oxygen saturation but that this variable in particular should be QCed due to biofouling so either we omit it or we include it but then definitely need to add processing_level for the dataset of NO QC or something similar (we should add that anyway).

Comment (knl88) [reply dscreator/datasets/timeseries/sios.py:114]: We can update the dataset with additional parameters later, would be nice to just get it setup to start with.  For processing level we should try to use one of mets keywords [here](https://vocab.met.no/mmd/en/page/?uri=https%3A%2F%2Fvocab.met.no%2Fmmd%2FOperational_Status%2FExperimental) , perhaps we start with experimental. 

Perhaps we can remove conductivity to start with, it is defined [here](https://github.com/NIVANorge/datasetcreator/blob/45bc3a19b617cc78af69e39776da6dd898e66a2b/dscreator/main.py#L40) and you need to swap the other variables with the calibrated once:)

Comment (IvanaH8) [reply dscreator/datasets/timeseries/sios.py:97]: Variables updated for calibrated values, we are keeping conductivity and adding fDOM (calibrated) for now as well after discussion with Sabine who gave a green light for 2023-24 data with disclaimer that the data are not completely QCed

Comment (IvanaH8) [reply dscreator/datasets/timeseries/sios.py:102]: I am also adding a comment comment = "This dataset has not been completely quality controlled"

**Pull request merged**